trino-aws-proxy-plugin-toolkit/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Why not put this in the `spi` module? Is there a benefit for a separate module?

Reply:
I initially though this might be bloated for the SPI module as it depends on some airlift components. 
Now I realise the core has those dependencies anyway so are provided to plugins regardless of what the SPI module does.

I'll move these back into the SPI module